internal/graph/graph.go

```diff
@@ -246,6 +246,23 @@ func CloneLinkerGraph(
 		}
 	}
 
+	if codeSplitting {
```

Comment:
Not essential to this PR 🤷 It does improve HMR, so it's good to get it out there as an extra incentive to try focus mode.